ocp-resources/e2e/spo-install.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openshift-security-profiles
+labels:
+  openshift.io/cluster-monitoring: "true"
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: security-profiles-operator
+  namespace: openshift-security-profiles
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: security-profiles-operator-sub
+  namespace: openshift-security-profiles
+spec:
+  channel: release-alpha-rhel-8
+  installPlanApproval: Automatic
+  name: security-profiles-operator
+  source: redhat-operators
```

Comment:
The FIO e2e remediation uses an upstream catalog source, but I don't think we have one of those for SPO (or any automation to produce it).

We might have to keep that in mind if this fails for some reason due to repository access.